pkg/apis/stable/v1alpha1/gameserverallocation.go

```diff
@@ -26,6 +26,9 @@ const (
 	GameServerAllocationAllocated GameServerAllocationState = "Allocated"
 	// GameServerAllocationUnAllocated when the allocation is unsuccessful
 	GameServerAllocationUnAllocated GameServerAllocationState = "UnAllocated"
+	// GameServerAllocationContention when the allocation is unsuccessful
+	// because of contention
+	GameServerAllocationContention GameServerAllocationState = "Contention"
```

Comment:
Do we want to document this somewhere? Maybe add a short line to https://agones.dev/site/docs/getting-started/create-fleet/#gameserverallocation , after "The state value will tell if a GameServer was allocated or not. If a GameServer could not be found, this will be set to UnAllocated." ?

Maybe do a line after that about what Contention means, and mark it to publish at 0.9.0 ?

https://agones.dev/site/docs/contribute/

Reply:
I am adding this line : 

> If there are too many concurrent requests ovewhelmed the system, some the `state`  will be set to 
> {{% feature publishVersion="0.9.0" %}}`Contention`{{% /feature %}} even though there are available `GameServer`.

Let me know what you think.

Reply:
That sounds perfect, I would put the `{{% feature publishVersion="0.9.0" %}}` block around the whole sentence. :+1: